app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+    has_many :posts
+    has_many :comments
```

Comment:
Hey add dependent: destroy, so the posts and comments are removed when the user who made them is removed. do that do posts also so comments are destroyed when post are.